File: benchmarks/cpu/Fib.java
public class Fib {
    static long fib(long n) {
        if (n <= 1) return n;
        return fib(n-1) + fib(n-2);
    }
    
    public static void main(String[] args) {
        System.out.println("fib(35) = " + fib(35));
    }
}

File: benchmarks/cpu/fib.c
#include <stdio.h>

long fib(long n) {
    if (n <= 1) return n;
    return fib(n-1) + fib(n-2);
}

int main() {
    printf("fib(35) = %ld\n", fib(35));
    return 0;
}
